Post/tests.py

```diff
@@ -149,3 +150,83 @@ def test_delete_post(client):
     )
 
     assert response.status_code == 302
+
+
+@pytest.mark.django_db
```

Comment:
You could use a fixture of create user every user,
create dictionary and parse it to fixture. 
see this pseudo 

@pytest.fixture()
def create_user():
    def _create_user(*args, **kwargs):
        return (args, kwargs)
        return new_user = User.objects.create_user( kwargs)
    return _create_user


def test_example(create_user):
    first_case = create_user({'username':'Nevo', 'email':'Test23@gmail.com', 'password':'password2233'})

Reply:
Thanks.
I don't know exactly how to do it-  I am working on it.

Reply:
@Yarboa 
We don't understand this example and what is the point, the 3 return is a row doesn't make any sense to us.
create_user takes 0  arguments but we send a dictionary.

Reply:
> @Yarboa
> We don't understand this example and what is the point, the 3 return is a row doesn't make any sense to us.
> create_user takes 0 arguments but we send a dictionary.

Try to print the kwargs

Reply:
@Yarboa,
I changed it as you asked, is it good?

Reply:
Almost :)
Did you think to do the same for create post?

Reply:
@Yarboa I did :)